mongodb-queue.js

```diff
@@ -265,3 +266,25 @@ Queue.prototype.done = function(callback) {
         callback(null, count)
     })
 }
+
+Queue.prototype.cancel = function(id, callback) {
+    var self = this
+
+    var query = {
+        _id     : new ObjectID(id),
+        deleted : null,
+        visible : { $lte : now() },
+    }
+    var update = {
+        $set : {
+            deleted : now(),
+        }
+    }
+    self.col.findOneAndUpdate(query, update, { returnOriginal : false }, function(err, msg, blah) {
```

Comment:
Yes, I was going to but I noticed the other callbacks were using it so I wanted to be consistent with the previous code. But I'll remove it, and the other unused pre-existing one if that's ok?